client.js

```diff
@@ -14,18 +14,18 @@ logger.level = 'debug';
 
 // Initialize Discord Client
 var bot = new Discord.Client();
-
-
+const TOKEN = settings.discord.token;
+const COMMAND_CHANNEL = settings.discord.command_channel;
 
 bot.on('ready', () => {
     logger.info(`Logged in as ${bot.user.tag}!`);
 });
 
 bot.on('message', function (message) {
 
-    // ignore bots and self and chat rooms
+    // ignore bots and non-command rooms
     // console.log(message);
-    if (message.author.bot || message.channel.id === '675891646235279386' || message.channel.id === '673793154729771028' || message.channel.id === '672512233435168784'){
+    if (message.author.bot || message.channel.id !== COMMAND_CHANNEL){
```

Comment:
This actually does not allow direct messages, think we will have to do it the other way unless discord.js has a meessage.channel.type prop we can check

Reply:
Fixed using type.